requirements.txt

```diff
@@ -0,0 +1,8 @@
+-e git+http://github.com/openaustralia/scraperwiki-python.git@morph_defaults#egg=scraperwiki
```

Comment:
That bit is necessary for it to work on morph.io, yes. I was wondering why yours was working without a requirements.txt file, but it must have something to do with being migrated from ScraperWiki, like you said.

I'm a little confused as to how your ScraperWiki scraper is still running, actually. I thought they stopped providing that service a few years ago? Anyway, morph.io's API is identical from what I can tell. Both use SQL queries. You'd probably just have to change the URL (and update your JSON object to support the new data). It would be fine to make a separate requirements.txt file as well, though.